Fixed #1564 Clip job cancel for multiple jobs fixed

bobi7 opened the pull request:
ref #1564

Commit:
fixed #1564. newly created folder on dataset import cancel will get removed

Commit:
Fixed #1564 Clip job cancel for multiple jobs working now

Commit:
Merge branch 'hoot2x' into 1564

brianhatchl commented on modules/Hoot/ui/modals/importDataset.js:
```diff
@@ -369,11 +369,17 @@ export default class ImportDataset {
+                        await Hoot.api.deleteFolder( folderId );
+                        await Hoot.folders.removeFolder( folderId );
+                    }
+
```
We talked about how this would be better done within the services.  If we moved folder creation to within a job command we could have that occur after successful clipping and if it was canceled the new output folder would never be created so we would not have to clean it up at all.  However, the services don't have a mechanism for passing input between job commands, so the folder id created by the AddFolder command would not be available to the UpdateParent command that moves output maps underneath said folder.
So we'll keep this way for now as it should run immediately after canceling.

bobi7 replied:
Note: moved this to https://github.com/ngageoint/hootenanny/issues/3412

brianhatchl reviewed (approved):
Looks good, I actually thought this was adding Cancel for multi-import, but I can do that next and will create a ticket.

brianhatchl commented:
I see it's already there #1563

brianhatchl committed:
fix jobId property name case

we should really make these consistent across service repsonses
because requests that submit jobs get 'jobid' returned
while he job resource that provides status returns 'jobId'
